advanced/sr_gui_biotac/src/sr_gui_biotac/biotac.py

```diff
@@ -284,36 +283,39 @@ def __init__(self, context):
         self._nb_electrodes = self._nb_electrodes_biotac
         self.assign_electrodes(self._nb_electrodes)
 
-        ui_file = os.path.join(rospkg.RosPack().get_path('sr_gui_biotac'),
-                               'uis', 'SrGuiBiotac.ui')
-        loadUi(ui_file, self._widget)
-        self._widget.setObjectName('SrBiotacUi')
-
-        self.timer = QTimer(self._widget)
-        # self._widget.connect(self.timer, SIGNAL("timeout()"),
-        #                     self._widget.scrollAreaWidgetContents.update)
-        self.timer.timeout.connect(self._widget.scrollAreaWidgetContents.update)
-        self._widget.scrollAreaWidgetContents.paintEvent = self.paintEvent
+        if rqt_plugin:
+            ui_file = os.path.join(rospkg.RosPack().get_path('sr_gui_biotac'),
+                                   'uis', 'SrGuiBiotac.ui')
+            loadUi(ui_file, self._widget)
+            self._widget.setObjectName('SrBiotacUi')
+
+            self.timer = QTimer(self._widget)
+            # self._widget.connect(self.timer, SIGNAL("timeout()"),
```

Comment:
Might this snippet be used later, or can it be deleted?

Reply:
Deleted :)